packages/frontity/src/cli/index.ts

```diff
@@ -56,14 +49,16 @@ program
   .command("create [name]")
   .option("-t, --typescript", "Adds support for TypeScript")
   .option("-c, --use-cwd", "Generates the project in the current directory.")
+  .option("-n, --no-prompt", "Skips prompting the user for options")
   .description("Creates a new Frontity project.")
-  .action(create);
+  .action((name, { ...args }) => create({ name, ...args }));
```

Comment:
It's not ideal that we have to remap the arguments here, but there's a mismatch between the proposed API and the callback args from `commander`.